src/MidnightExample/Ledger.lagda

```diff
@@ -95,15 +96,17 @@ instance
 \AgdaTarget{Header, slotNo, blockNo, blockHash, prev, nodeId, Block, header, body, blockPoint, computeBlockHash, addBlockHash}
 \begin{code}
 record Header : Type where
-  field  slotNo     : ℕ
-         blockNo    : ℕ
-         blockHash  : Hash
-         prev       : ℕ
-         nodeId     : ℕ
+  field
+    slotNo     : ℕ
+    blockNo    : ℕ
+    blockHash  : Hash
+    prev       : ℕ
+    nodeId     : ℕ
 
 record Block : Type where
-  field  header  : Header
-         body    : List Tx
+  field
+    header  : Header
+    body    : List Tx
```

Comment:
I uniformly applied the same convention to indentation of field names.

Reply:
LGTM